tests/routes/accountOpeningRequest.spec.ts

```diff
@@ -105,4 +122,67 @@ describe("Account Opening Request", () => {
       expect(res.send.getCall(0).args[0].id).to.equal(accountOpeningRequestId);
     });
   });
+
+  describe("create Account Opening Request (business)", () => {
+    before(async () => {
+      const req = mockReq({
+        body: {
+          customer_id: businessId,
+          customer_type: CustomerType.BUSINESS,
+          product_name: "CURRENT_ACCOUNT_FREELANCER_GERMANY",
+          account_type: AccountType.CHECKING_SOLE_PROPRIETOR,
+          account_currency: "EUR",
+          account_purpose: "primary",
+          account_bic: process.env.SOLARIS_BIC,
+        },
+      });
+
+      await accountOpeningRequestAPI.createAccountOpeningRequest(req, res);
+
+      business = await db.getBusiness(businessId);
+    });
+
+    it("should return successful response", () => {
+      expect(res.status.calledWith(201)).to.be.true;
+    });
+
+    it("should create an accountOpeningRequest on Business entity", async () => {
+      expect(business.accountOpeningRequests).to.have.length(1);
+      expect(business.accountOpeningRequests[0]).to.be.an("object");
+    });
+
+    it("should create an account for business", async () => {
+      expect(business.account).to.be.an("object");
```

Comment:
please also test if account of proper type is created

Reply:
consider it done